lib/wrest/native/delete.rb

```diff
@@ -9,15 +9,15 @@
 
 module Wrest::Native
   class Delete < Request
-    def initialize(wrest_uri, parameters = {}, headers = {}, options = {})
+    def initialize(wrest_uri, body = '', headers = {}, parameters = {} ,options = {})
```

Comment:
I would expect body to be an hash as well, why did you define it as an empty string for default?

Reply:
@andreorvalho  I'm just following the same pattern as the other http method with a body in this file like https://github.com/Vivino/wrest/blob/8747cc8988e2e9fdc0024d4363e8e86bb245fc0b/lib/wrest/uri.rb#L125 etc

Reply:
yea I see. @corthmann do you know why is it like this?